CHANGELOG.rst

```diff
@@ -1,9 +1,11 @@
 Unreleased
 ----------
-* Drop support for Wagtail versions 4.2 and 5.0 (@katdom13)
-* Support for Wagtail 5.2 (@katdom13)
+* Drop support for Wagtail < 5.2 (@katdom13)
+* Add support for Wagtail 6.0 (@katdom13)
+* Add support for Python 3.12 (@katdom13)
 
-* Update for Wagtail 5.1
+* Drop support for Wagtail versions 4.2 and 5.0 (@katdom13)
+* Support for Wagtail 5.1 and 5.2 (@katdom13)
```

Comment:
Should this now just be support for Wagtail 5.2 as 5.1 got dropped?

Reply:
Whoops, that correct.
Those were the last unreleased notes from the last round of upgrade. They were still under "Unreleased" because we still have an outstanding Wagtail <= 5.2 PR to the maintainer.

Removed those.
Thanks!